querybook/webapp/components/DataTableViewColumn/DataTableColumnCard.tsx

```diff
@@ -69,6 +86,11 @@ export const DataTableColumnCard: React.FunctionComponent<IProps> = ({
                                 />
                             </KeyContentDisplay>
                         )}
+                        {columnTags && columnTags.length > 0 && (
+                            <KeyContentDisplay keyString="Tags">
```

Comment:
better if you do
```
    const tagsDOM = (columnTags || []).map((tag) => (
                <TableTag tag={tag} readonly={true} key={tag.id} mini={true} />
            ));

   tagsDOM && (
      <KeyContentDisplay keyString="Tags"><div className="DataTableTags flex-row">{tagsDOM}</div>
      </KeyContentDisplay>

  )

```

Reply:
tagsDom is not falsey when columnTags is []

Reply:
oh then tagsDOm.length